packages/waku/src/lib/builder/build.ts

```diff
@@ -89,8 +89,8 @@ const analyzeEntries = async (entriesFile: string) => {
     plugins: [rscAnalyzePlugin(commonFileSet, clientFileSet, serverFileSet)],
     ssr: {
       resolve: {
-        conditions: ['react-server', 'workerd'],
-        externalConditions: ['react-server', 'workerd'],
+        conditions: ['react-server', 'worker'],
+        externalConditions: ['react-server', 'worker'],
```

Comment:
I don't know which one is picked in this case.
https://github.com/facebook/react/blob/5c607369ceebe56d85175df84b7b6ad58dd25e1f/packages/react-server-dom-webpack/package.json#L52

I'm afraid it's working unexpectedly.

Reply:
I think it will be: react-server -> node -> default

according to the nodejs spec

Reply:
If we merge this PR also which uses conditions, it will be fine

https://github.com/dai-shi/waku/pull/392